src/styles/todo.scss

```diff
@@ -65,21 +65,21 @@
       color: #af5b5e;
     }
   }
-  
+
   &:hover &__remove {
     opacity: 1;
   }
 
   &__title-field {
     width: 100%;
     padding: 11px 14px;
-  
+
     font-size: inherit;
     line-height: inherit;
     font-family: inherit;
     font-weight: inherit;
     color: inherit;
-  
+
```

Comment:
There is an extra whitespace added. It is not a major issue but it's good to keep the code clean.